### PR TITLE
[FIX] #22 Search transition and color

### LINKED ISSUE #22: Search transition and color
when hovering the mouse on search icon in the header the icons blinks (it  has an animation that changes it's opacity)
I want that to be disabled
Screenshot 2024-09-06 171933

### PR DESCRIPTION
**Pull request by MiteshDalvi**
I have resolved the search icon blinking animation. Actually there was an 3s infinite animation code written in header.scss file due to which the issue was coming. The search icon now has static animation and while hovering, it change to pink color without changing its opacity
fixes #22

https://github.com/user-attachments/assets/7204bf8f-4d0b-4d03-a7f5-e29a1c548bae

.

### CODE REVIEW

File: scss/_header.scss [hunk 1]
```diff
@@ -660,7 +660,7 @@
     transform 0.4s ease, 
     filter 0.4s ease, 
     box-shadow 0.4s ease;
-    animation: gradient-pulse 3s infinite;
+    animation: none;
     @keyframes gradient-pulse {
```
Comment (successbyte) [inline scss/_header.scss:664]: Please delete the keyframe related to the animation and check if its not used in another place 
thanks

**Review by successbyte — COMMENTED** [review]
Please delete the keyframe related to the animation and check if its not used in another place
thanks

**Review by successbyte — CHANGES_REQUESTED** [review]
Please delete the keyframe related to the animation and check if its not used in another place
thanks

**Comment by MiteshDalvi** [comment]
I have committed the changes.

**Review by successbyte — APPROVED** [review]
It's great thanks

**Comment by successbyte** [comment]
Please give the repo a star 
and invite your friends if they are interested in their first contribution.
thanks

**Pull request merged**